config/config_slac_testing.yaml

```diff
@@ -66,3 +66,8 @@ comCamBinning: 8
 # # paths for serving TS8 metadata at SLAC
 comCamMetadataPath: '/sdf/home/m/mfl/u/rubintv/LSSTComCam/sidecar_metadata'
 comCamMetadataShardPath: '/sdf/home/m/mfl/u/rubintv/LSSTComCam/sidecar_metadata/shards'
+
+
+# summit-like configs
+tmaMetadataPath: '/sdf/home/m/mfl/u/rubintv/tma/sidecar_metadata'
```

Comment:
Since these are similar to those for ComCam, I assume there's a ticket to migrate these to a user-agnostic location already.

Reply:
Not a ticket, at such, but yes, it's a bit piece of tech-debt in RA. I'll file it now though, for sorting out LocationConfig class entirely.

Reply:
DM-42912